### PR TITLE
Fixed some broken things in profiles

### PR DESCRIPTION
**Pull request by DarkPurple141**
* Profile fixes
* Amend user mapper
* Redundant styles
* aria-attributes

### CODE REVIEW

File: frontend/src/utils/api/user.js [hunk 1]
```diff
@@ -14,8 +14,12 @@ export function userMapper({
   joined,
   ...rest
 }) {
+  if (!joined) {
+    return
```
Comment (NunoDasNeves) [inline frontend/src/utils/api/user.js]: return nothing?

Comment (NunoDasNeves) [reply frontend/src/utils/api/user.js]: shouldn't it return `rest` or `{ ...rest }` ?

Comment (NunoDasNeves) [reply frontend/src/utils/api/user.js]: just to stop errors happening idk

Comment (DarkPurple141) [reply frontend/src/utils/api/user.js]: Hmm. Yeah I dunno. To me it implies the user doesn't exist, and other fields will also error.

Comment (DarkPurple141) [reply frontend/src/utils/api/user.js]: `joined` is compulsory I'm pretty sure.

**Pull request merged**